sdk/batch/batch-rest/README.md

```diff
@@ -6,7 +6,8 @@ Azure Batch provides Cloud-scale job scheduling and compute management.
 
 Key links:
 
-TODO: Add links back when the package is published
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/batch)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/batch?view=azure-node-preview)
```

Comment:
Do we need ?view=azure-node-preview in the link? It seems to redirect there when I don't have it.